Makefile

```diff
@@ -314,6 +312,10 @@ clean:
 	echo "-pc tools"
 	$(MAKE) -C pc clean
 
+realclean: clean
```

Comment:
realclean -> clean
clean -> mostlyclean
https://www.gnu.org/software/make/manual/html_node/Goals.html

Reply:
I disagree. We're talking about deleting external **source** files. We not talking about deleting **compiled** files.

From the link you provided:

> mostlyclean
> Like ‘clean’, but may refrain from deleting a few files that people normally don’t want to recompile. For example, the ‘mostlyclean’ target for GCC does not delete libgcc.a, because recompiling it is rarely necessary and takes a lot of time.
> 

> distclean
> realclean
> clobber
> Any of these targets might be defined to delete more files than ‘clean’ does. For example, this would delete configuration files or links that you would normally create as preparation for compilation, even if the makefile itself cannot create these files.
> 

Which is exactly what I think we want:
- clean: delete all compiled files
- realclean: delete more, like the entire git repository